plugins/services/src/js/components/forms/NetworkingFormSection.js

```diff
@@ -556,7 +561,7 @@ class NetworkingFormSection extends mixin(StoreMixin) {
       })
       .getItems()
       .map((virtualNetwork, index) => {
-        return (
+        return mesosContainer && virtualNetwork.text === "dcos6" ? null : (
```

Comment:
Could you do this with a `filterItems` instead? I think it would be the clearer and more functional approach

```diff
+ .filterItems(item => !mesosContainer || virtualNetwork.text !== "dcos6")
  .getItems()
  .map((virtualNetwork, index) => {
-    return mesosContainer && virtualNetwork.text === "dcos6" ? null : (
+    return (